cluster/concourse.yml

```diff
@@ -48,7 +48,7 @@ instance_groups:
 
 - name: db
   instances: 1
-  azs: [z1]
```

Comment:
I don't think this change should be added as part of this. Seems like a separate discussion. This will also affect any existing deployments (like ours).